android/app/src/main/AndroidManifest.xml

```diff
@@ -29,23 +29,24 @@
       <activity
         android:name=".bootsplash.BootSplashActivity"
         android:theme="@style/BootTheme"
-        android:launchMode="singleTask">
+        android:launchMode="singleTask"
+        android:exported="true">
```

Comment:
```suggestion
        android:exported="false">
```

Reply:
The value for `android:exported` must be set to true, otherwise the links won't open the app.

According  to [Android docs](https://developer.android.com/about/versions/12/behavior-changes-12#exported):

> If the app component includes the [LAUNCHER](https://developer.android.com/reference/android/content/Intent#CATEGORY_LAUNCHER) category, set android:exported to true. In most other cases, set android:exported to false.

Our app component includes the LAUNCHER category

https://github.com/tomivs/App/blob/e55f74de55f087e1b0fd77afdc1c6d2356ccb0b2/android/app/src/main/AndroidManifest.xml#L37

Reply:
Ahh! I see.